spec/system/admin/admin_manages_profile_fields_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "rails_helper"
+
+RSpec.describe "Admin manages profile fields", type: :system do
+  let(:admin) { create(:user, :super_admin) }
+  let!(:profile_field_group) { create(:profile_field_group, name: "Delete Me!") }
+  let!(:profile_field) { create(:profile_field, profile_field_group: profile_field_group, label: "Delete Me Too!") }
+
+  before do
+    Flipper.enable(:profile_admin)
```

Comment:
sorry, can we replace this with `FeatureFlag.enable` ? Even better, use `allow(FeatureFlag).to receive(:enable).and_return(true)` so that it won't even write to the DB and be a little faster?

See #11386

Reply:
Great point! I tried using `allow()`initially, but it doesn't seem to work. I'll definitely use the wrapper though!